### DISCUSSION TITLE
如何解决黄金右键与快捷键拓展冲突问题

### DISCUSSION THREAD

**Discussion by BlueFunny19** [the1812/Bilibili-Evolved#1490] [提问 / Q&A] [answered]
所有视频播放网页因为"→"被快捷键拓展功能占用而导致黄金右键(脚本链接:https://greasyfork.org/zh-CN/scripts/396467-%E9%BB%84%E9%87%91%E5%8F%B3%E9%94%AE)无法使用

**Comment by Geeyun-JY3** [comment]
论鱼和熊掌如何兼得……

**Comment by the1812** [comment, marked as answer]
可以删除脚本的 `→` 快捷键啊

**Reply by BlueFunny19** [reply]
我知道这个东西，但是我不知道"→"被定义的名称啊（笑哭）

**Reply by BlueFunny19** [reply]
我甚至不知道右键加速被定义成了什么名字
我觉得下个版本确实该加个这个东西了，我都不知道有啥可以自定义的功能以及对应按键的名称（汗）

**Reply by the1812** [reply]
图形界面可能还得等段时间, 现在只能从代码上去改

**Reply by the1812** [reply]
```js
bilibiliEvolved.settings.customKeyBindings = { jumpForward: '' }
```

**Reply by BlueFunny19** [reply]
thx very much
